lib/slimmer/processors/layout_header_manager.rb

```diff
@@ -1,12 +1,12 @@
 module Slimmer::Processors
-  class AccountsShower
+  class LayoutHeaderManager
     def initialize(headers)
       @headers = headers
     end
 
     def filter(_src, dest)
       header_value = @headers[Slimmer::Headers::SHOW_ACCOUNTS_HEADER]
-      layout_header = dest.at_css(".gem-c-layout-header")
+      layout_header = dest.xpath("//*[not (@id='wrapper')]/header[@class='gem-c-layout-header']")
```

Comment:
So what's behind this selector then?
Does ours use an `id=wrapper` and the classic GOV.UK one does not?